kernels/lit.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+#
+# To build the kernel:  ./kernels/lit.sh
+# To remove the kernel: ./kernels/lit.sh remove
+#
+# This scripts will create a ipython kernel named $ENVNAME
+
+ENVNAME=lit_kernel
+KERNEL_DISPLAY_NAME="LIT kernel"
+
+source ~/.bashrc
+
+# Cleaning up the kernel and exiting if first arg is 'remove'
+if [ "$1" == "remove" ]; then
+  echo Removing kernel $ENVNAME
+  jupyter kernelspec remove $ENVNAME
+  conda env remove -n $ENVNAME -y
+  exit 0
+fi
+
+conda create -n $ENVNAME python=3.10.14 -y
+conda activate $ENVNAME
+
+# Install packages using a pip local to the conda environment.
+conda install pip
```

Comment:
You could add a -y flag here to install quietly

Reply:
probably -q? Added it